style.css

```diff
@@ -16,10 +16,106 @@ img {
   max-width: 100%;
 }
 
-.section {
+.banner {
+  padding: 2rem;
+  background: #276583;
+  color: #fefefe;
+  line-height: .5rem;
+
+}
+
+.navbar__group-left {
+  float: left;
+}
+
+.navbar__group-right {
+  float: right;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+  margin: -0.5rem -1rem;
+}
 
+.nav-list__item {
+  margin: .5rem 1rem;
+}
+
+.logo {
+  margin: 0;
+  font-size: 1.5rem;
+}
+
+
+/*Grid*/
+
+.section {
+  padding: 1rem;
 }
 
 .container {
+  max-width: 65rem;
+  margin: 0 auto;
+}
+
+.grid {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+/*Card*/
+
+.grid__item {
+   margin: 2rem;
+   flex-basis: calc(100% / 2 - 2rem - 2rem);
+   flex-grow: 1;
+}
+
+.card {
+  background: #ffae4f;
+  border: solid 2px;
+  border-style: outset;
+  margin: 2;
+  padding: 0;
+  min-width: 8rem;
+  position: relative;
+
+}
+
+.top-card {
+  position: relative;
+}
+
+.card__title {
+  padding: .5rem;
+  margin: 1rem;
+}
+
+.card__pic {
+  max-width: 3rem;
+  border-radius: 50%;
+  border: solid grey 1px;
+  position: absolute;
+  margin: 0;
+  top: 1rem;
+  left: -2rem;
+  transform: translateY(-50%);
+}
+
+.card__tag {
+  position: absolute;
+  background: #276583;
+  color: white;
+  padding: .8rem;
+  margin: 0;
+  top: -2rem;
```

Comment:
Make sure to use "non-guessing" units for positioning values and stick to things like 0, 50%, and 100% along with similar values for transform with translate.

When the card tag runs to different lines this is no longer centered on the top edge.